core/src/banking_stage.rs

```diff
@@ -1351,38 +1351,40 @@ impl BankingStage {
         gossip_vote_sender: &ReplayVoteSender,
         qos_service: &QosService,
     ) -> ProcessTransactionBatchOutput {
-        let ((transactions_qos_results, cost_model_throttled_transactions_count), cost_model_time) =
-            Measure::this(
-                |_| {
-                    let tx_costs = qos_service.compute_transaction_costs(txs.iter());
+        let (
+            (transactions_qos_results, cost_model_throttled_transactions_count, transaction_costs),
+            cost_model_time,
+        ) = Measure::this(
```

Comment:
nit: `Measure::this` doesn't really help to make code cleaner when it has to pass out many results. May as well just call `Measure::start` and `stop`.